### ISSUE TITLE
[Suggestion] More uses for Alternator

### ISSUE THREAD

**Issue by Koltanas** [FerroO2000/Immersive-Tech#27]
The new update has divided the Steam Turbine into two separate maschines, something I actually like.
But currently there is not other option than to directly connect them, making the update pretty pointless, except that it kinda brakes savegames.
I see three basic options for powering a larger generator:
-A steam based engine. This is already impelmented with the Steam Turbine, could add a piston steam engine, but I don't think such a thing would have a place.
-Internal combustion, so either a diesel engine, which is already in Immersive Engineering, or a gas turbine, which would need a new fuel source
-Rotatory power from wind or water. The first one would need a large windmill, I don't know if such a thing would fit into the mods general theme. The later would need a water turbine, which unlike the waterwheel from immersive engineering has to have water flowing through it and besides that I don't know how one would make the structure appropriate complicated for the power output with the means minecraft offers.

Besides other means to power the alternator, it would be nice if one could displace the alternator and the turbine and connect them with axles and gearboxes.

**Comment by FerroO2000** [comment]
Alternator is ispired by Rotarycraft and I have already prepared to add some rotors, a gerabox which can sort the mechanical energy

**Comment by Koltanas** [comment]
Nice to hear that. Is there some long term goal for the mods theme?

**Comment by FerroO2000** [comment]
Yes. I will end the energy production part and I will begin another one based on items transport with cable cars

**Comment by Koltanas** [comment]
That sounds like some project. What use will those cable cars have?
